src/App/Fossa/Container/Sources/DockerTarball.hs

```diff
@@ -72,6 +72,7 @@ import Path.Internal (Path (Path))
 import Srclib.Converter qualified as Srclib
 import Srclib.Types (SourceUnit)
 import Strategy.ApkDatabase qualified as Apk
+import Strategy.DpkgDatabase qualified as Dpkg
```

Comment:
This qualification is a bit confusing to me because there's also a module in the tree called `Dpkg`. I would spell this out all the way.